.github/ISSUE_TEMPLATE/nomination.md

```diff
@@ -17,8 +17,9 @@ with the lowest [GitHub Permission Level] of Read. For example, Contributors
 with Read permission do not have the permission to add reviewers to a pull
 request.
 
-Use this template to nominate a GitHub user for an elevated permission level in
-the Contributor role.
+Use this template to nominate a GitHub user for the Triage permission level,
```

Comment:
Why remove "contributor role" completely? I would keep this, maybe:

> Use this template to nominate a GitHub user for the contributor role with Triage permission level,....

Reply:
Fixed